<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTableWidgetDemoClass</class>
 <widget class="QMainWindow" name="QTableWidgetDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTableWidgetDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="btnInitHead">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>设置表头</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSetRow">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>设置行数</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnInitData">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>初始化表格数据</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnInsertRow">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>插入行</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnAddRow">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>添加行</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDelRow">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>删除行</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnAutoColW">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>160</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>自动调节列宽</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnAutoRowH">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>自动调节行高</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnReadToText">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>读取表格内容到文本</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cbEditable">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>表格可编辑</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="cbSpacingColor">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>220</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>间隔行底色</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="cbShowRowHead">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>显示行表头</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="cbShowColHead">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>250</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>显示列表头</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="rbSelRow">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>行选择</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="rbSelCell">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>280</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>单元格选择</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="sbRowCount">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTableWidget" name="tableWidget">
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>2</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>3</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>4</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>5</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>6</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>A</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>B</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>C</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>D</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>E</string>
            </property>
           </column>
          </widget>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>754</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QTableWidgetDemo.qrc"/>
 </resources>
 <connections/>
</ui>
